src/components/datatable.component.ts

```diff
@@ -386,6 +387,16 @@ export class DatatableComponent implements OnInit, DoCheck, AfterViewInit {
   @Input() selectCheck: any;
 
   /**
+   * A function you can use to check whether you want
+   * to show the checkbox for a particular row based on a criteria. Example:
+   *
+   *    (row) => {
```

Comment:
Can you update this comment?

Reply:
I already did it, this diff is not up to date. Look: https://github.com/swimlane/ngx-datatable/pull/1056/files